Sources/ENFoundation/Extensions/NSAttributedString+Extension.swift

```diff
@@ -10,6 +10,9 @@ import UIKit
 
 public extension NSAttributedString {
 
+    static let accessibilityListIndex = "index"
```

Comment:
Maybe wrap these static strings in a "AccessibilityTextCustomValue" enum to give them some context?

Reply:
resolved this myself